src/main/java/com/monitorjbl/xlsx/impl/StreamingSheetReader.java

```diff
@@ -39,6 +40,7 @@
   private final StylesTable stylesTable;
   private final XMLEventReader parser;
   private final DataFormatter dataFormatter = new DataFormatter();
+  private final Set<Integer> hiddenColumns = new HashSet<>();
```

Comment:
I'm not sure why this needs to be here. This information only applies to Rows, correct?